packages/core/admin/ee/server/src/controllers/admin.ts

```diff
@@ -6,6 +6,16 @@ import { env } from '@strapi/utils';
 import { getService } from '../utils';
 
 export default {
+  // NOTE: Overrides CE admin controller
+  async getProjectType() {
+    const flags = strapi.config.get('admin.flags', {});
+    try {
+      return { data: { isEE: strapi.EE, features: ee.features.list(), flags } };
+    } catch (err) {
+      return { data: { isEE: false, features: [], flags } };
+    }
+  },
+
```

Comment:
I am not completely sure if this is the right solution, but if it is, we will be able to completely avoid the circular dependency with strapi/strapi.
I am investigating now